Running build_fast.sh provides a 1-step build process

jhsu802701 opened the pull request:
* The build_fast.sh script sets up everything and runs tests, which all pass.
* build_fast.sh script is based on the settings and commands in the Travis CI environment.
* Selected Capybara tests in spec/features/adoption_form_spec.rb are allowed up to a 20-second wait time to eliminate intermittent test failures.
* Creating the PostgreSQL extension postgis requires the packages postgresql-9.4-postgis-2.1 and postgresql-9.4-postgis-scripts (now provided in the custom Adopt-a-Tree Docker image).
* json gem upgraded, because "bundle install" refused to install version 1.8.1
* The mailcatcher.sh script combined with the revised config/environments/development.rb setup allows you to simulate the user experience with emails.
* The server.sh script starts up the local server and makes the web pages of this app available on the host system.

wbushey commented:
Great work @jhsu802701. I have intermittent internet access, so it will probably be a couple of days until I really go through this/try it. One immediate bit of feedback: can you add directions to the README?

jhsu802701 commented:
I have updated the setup procedure in the README.md file.  Additionally, I have added the test_app.sh script, which runs "bundle install", the database migrations, and the testing.

jhsu802701 commented:
I added a section about switching from Dash to Bash in README.md.  I struggled with getting sed commands to work in Dash and ended up just using Bash.
